zarr/core.py

```diff
@@ -397,7 +397,7 @@ def fill_value(self, new):
         self._flush_metadata_nosync()
 
     @property
-    def order(self):
+    def order(self) -> str:
```

Comment:
shouldn't this be `Literal["C", "F"]`?

Reply:
Ah thanks for catching, should be fixed now.